webapp/package.json

```diff
@@ -65,10 +66,13 @@
   "dependencies": {
     "core-js": "3.6.5",
     "mattermost-redux": "5.23.0",
+    "prop-types": "15.7.2",
     "react": "16.14.0",
+    "react-dom": "18.2.0",
```

Comment:
This is causing build errors for me. I removed it again and it built fine. There are some test failures, though, but I'm not sure if they are caused by this.

Reply:
@Snektron It's building fine for me with this dependency and falling if I remove it. Also, this dependency is being used in the code, so I don't think it can be removed. Can you please share some screenshots of the errors you are facing?

Reply:
Note that if we are going to install `react-dom`, it should always have the same version number as `react`. Not sure how npm let you do that otherwise

Reply:
@mickmister I have updated it in the last commit